packages/signature-controller/src/SignatureController.ts

```diff
@@ -386,6 +386,26 @@ export class SignatureController extends BaseControllerV2<
     );
   }
 
+  /**
+   * Called when a MMI wants to update the message status as signed.
```

Comment:
We don't want to mention MMI anywhere as the goal of all these abstractions is to decouple it from knowledge of MMI.

Reply:
Got it, will clean that up! 💪🏼